frontend/src/static/js/v_authorship.js

```diff
@@ -335,6 +375,14 @@ window.vAuthorship = {
     this.setInfoHash();
   },
 
+  beforeDestroy() {
+    window.removeHash('authorshipFileTypes');
```

Comment:
hmm, should here also remove `tabAuthor`, `tabRepo` and `authorshipIsMergeGroup` instead of removing in `main.js`?

Reply:
Good idea, didn't notice this. Would be good to centralize where authorship hashes are removed

I couldn't find a `authorshipIsMergeGroup` hash though, was this removed recently?

Reply:
`authorshipIsMergeGroup` was added recently in PR #1175. Look at `main.js`.

Reply:
Ahh my bad, thought I was on the latest already, thanks!

Reply:
updated, should we open an issue to investigate doing the same for the zoom tab?

Reply:
> updated, should we open an issue to investigate doing the same for the zoom tab?

Yes please do. I believe it will be much cleaner for the component themselves to remove their hashes.